test/system/700-play.bats

```diff
@@ -168,3 +168,48 @@ _EOF
     run_podman pod rm -t 0 -f test_pod
     run_podman rmi -f userimage:latest
 }
+
+@test "podman play --build --context-dir" {
+   if is_remote; then
+       skip "--build is not supported in context remote"
+   fi
```

Comment:
Shorter & sweeter: `skip_if_remote "--build is not supported in remote context"`

Reply:
I want to see if this passes, then I will make that change, and attempt to cd PODMAN_TMPDIR again.